custom/custom.go

```diff
@@ -0,0 +1,17 @@
+package custom
+
+import "github.com/ONSdigital/dp-api-clients-go/v2/dataset"
+
+func GenerateCustomTitle(dims []dataset.VersionDimension) string {
+	var title string
+	l := len(dims)
```

Comment:
We could calculate the last index only once and store it in a variable named accordingly for easier reading.
```suggestion
	lastIndex := len(dims) - 1
```